droidlet/interpreter/robot/tasks.py

```diff
@@ -519,6 +520,71 @@ def step(self):
     def __repr__(self):
         return "<ExamineDetectionStraightline {}>".format(self.target['label'])
 
+
+class ExamineDetectionCircle(TrajectorySaverTask):
```

Comment:
fine for now, but I would really rather this and the straight line one to be a single class with the path type in the task data

Reply:
Makes sense. Created an issue that I'll come back to in a hack session (https://github.com/facebookresearch/fairo/issues/906). Don't want to keep adding to this PR since it is becoming quite large.